src/Raven.Client/Documents/DocumentStore.cs

```diff
@@ -268,21 +268,31 @@ public override IDisposable DisableAggressiveCaching(string database = null)
         /// <summary>
         /// Subscribe to change notifications from the server
         /// </summary>
-        public override IDatabaseChanges Changes(string database = null)
+        public override IDatabaseChanges Changes(string database = null, string nodeTag = null)
         {
             AssertInitialized();
 
-            return _databaseChanges.GetOrAdd(database ?? Database, CreateDatabaseChanges);
+            return _databaseChanges.GetOrAdd(new RequestDestination
+            {
+                DatabaseName = database ?? Database,
+                NodeTag = nodeTag
+            }, CreateDatabaseChanges);
         }
 
-        protected virtual IDatabaseChanges CreateDatabaseChanges(string database)
+        internal virtual IDatabaseChanges CreateDatabaseChanges(RequestDestination node)
```

Comment:
rename `RequestDestination` to `DatabaseChangesOptions`

Reply:
done